src/i18n/en.ts

```diff
@@ -368,7 +368,9 @@ const en = {
         "term.metadata.altLabels.label": "Synonyms",
         "term.metadata.altLabels.placeholder":
             "Enter a new synonym and press 'Add'",
-        "term.metadata.altLabels.placeholder.text": "Add",
+        "term.metadata.altLabels.addButton.text": "Add",
+        "term.metadata.altLabels.addButton.title":
+            "Click to add the alternative label value",
```

Comment:
Why not simply "Click to add the synonym" (also, the UI speaks about synonyms, so it should be consistent)